app/controllers/contestant_projects_controller.rb

```diff
@@ -0,0 +1,14 @@
+class ContestantProjectsController < ApplicationController
+  def create
+    contestant_project = ContestantProject.new(contestant_project_params)
+    if contestant_project.save
```

Comment:
Nice additional error handling! We don't require this on ICs but it's great to practice when you have the time!